Chat/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Is node 14 still the reccomended version here?
https://github.com/Azure-Samples/communication-services-web-chat-hero